core/src/test/java/apoc/trigger/TriggerNewProceduresTest.java

```diff
@@ -532,4 +567,59 @@ public void testInstallTriggerInSystemDb() {
         }
     }
 
+    @Test
+    public void testEventualConsistency() {
+        LongStream.range(0, 100).forEach(i -> {
+            final String name = UUID.randomUUID().toString();
+            final String query = "UNWIND $createdNodes AS n SET n.count = " + i;
+    
+            // this does nothing, just to test consistency
+            sysDb.executeTransactionally("CALL apoc.trigger.install('neo4j', $name, 'return 1', {phase: 'afterAsync'})",
+                    map("name", UUID.randomUUID().toString()) );
+
+            // create trigger
+            sysDb.executeTransactionally("CALL apoc.trigger.install('neo4j', $name, $query, {phase: 'afterAsync'})",
+                    map("name", name, "query", query));
+            awaitTriggerDiscovered(db, name, query);
+            db.executeTransactionally("CREATE (n:Something)");
+
+            // check trigger
+            assertEventually(() ->
+                            db.executeTransactionally("MATCH (c:Something) RETURN c.count as count", Map.of(),
+                                    result -> {
+                                        final ResourceIterator<Object> count = result.columnAs("count");
+                                        return Objects.equals(count.next(), i);
+                                    }), 
+                    (v) -> v, 30L, TimeUnit.SECONDS);
+
+            testCall(db, "MATCH (c:Something) RETURN c.count as count",
+                    (row) -> assertEquals(i, row.get("count")));
+
+            // stop trigger
+            sysDb.executeTransactionally("CALL apoc.trigger.stop('neo4j', $name)",
+                    map("name", name, "query", query));
+            awaitTriggerDiscovered(db, name, query, true);
+
+            // this does nothing, just to test consistency
+            sysDb.executeTransactionally("CALL apoc.trigger.install('neo4j', $name, 'return 1', {phase: 'afterAsync'})",
+                    map("name", UUID.randomUUID().toString()) );
```

Comment:
I'm not sure how this test consistency? Same for the block of code from L.576-578

Reply:
Answered in the other pr too
```
for some reason by adding more triggers, 
the probability of making the test flaky seems higher 
(but anyway, it seems to affect only the embedded test instance)
```

Reply:
Did you forget to remove this test from the PR? It looks like the same test in another PR I reviewed yesterday (https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/3355) and I can't seem to see how it is testing apoc.triggers.show

Reply:
It seems so :(  However I did the rebase and removed it

Reply:
No worries :) Thanks for cleaning it up!